apps/api/.env.example

```
@@ -15,3 +15,12 @@ GOLF_GENIUS_RETRY_DELAY_BASE=1000
 # API runtime env examples
 NODE_ENV=development
 PORT=3333
+
+# Mail configuration
+ 
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)